python-backend/app/api/parties/party_appt/resources/mine_party_appt_resource.py

```diff
@@ -26,18 +26,20 @@ class MinePartyApptResource(Resource, UserMixin, ErrorMixin):
     @api.doc(params={'mine_party_appt_guid': 'mine party appointment serial id'})
     @requires_role_mine_view
     def get(self, mine_party_appt_guid=None):
+        relationships = request.args.get('relationships')
+        relationships = relationships.split(',') if relationships else []
```

Comment:
I think in theory the HTTP standard calls for the query params to look like `/mines?relationship=parties&relationship=permits` and then we can use `request.args.getlist('relationship')` but this exists in other parts. Just an FYI, not a blocker.

Reply:
I'm fine with either pattern. The comma-separated approach is used elsewhere for listed values, so I did that here. If it's not the pattern we want to use then I'll update this -- but I'll ask for team consensus around that first, since we already have one pattern in place